host/sgx/sgx_enclave_common_wrapper.c

```diff
@@ -125,11 +125,12 @@ static void _load_sgx_enclave_common_impl(void)
     {
         OE_CHECK(_lookup_function("enclave_create", (void**)&_enclave_create));
         /*
-         * NOTE: _enclave_create_ex is available only in newer PSW. Hence, it
-         * should not check for valid function pointer until all systems upgrade
-         * PSW.
+         * NOTE: enclave_create_ex() is available only in newer PSW. We should
+         * not check for valid function pointer until all systems upgrade to
+         * PSW version 2.14.1 or higher.
+         * Hence, directly use LOOKUP_FUNCTION() and not _lookup_function().
          */
-        _lookup_function("enclave_create_ex", (void**)&_enclave_create_ex);
+        _enclave_create_ex = LOOKUP_FUNCTION("enclave_create_ex");
```

Comment:
To make the trace more meaningful, can we still throw a warning/info level message if LOOKUP_FUNCTION returns NULL?

Reply:
@anakrish what do you think about a OE_TRACE_INFO with a message "enclave_create_ex not found in sgx_enclave_common.so.1. Need PSW version 2.14.1 or higher." ? This message will be logged every time _load_sgx_enclave_common() is called.

Reply:
Yes, INFO should be good. The message won't show up by default.